### PR TITLE
#163693759 FT-implement-Landing-page-components

### PR DESCRIPTION
**Pull request by ibraheemkabir**
**What does this PR do?**
This PR implements the components of the landing page of AH

**Description of Task to be completed**
- it Implements landing page components

**How should this be manually tested?**
* `npm test`
* Run `npm start` and open the `http://localhost:5000` in the browser

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/story/show/#163693759

Screenshots
screen shot 2019-02-12 at 6 44 21 pm

video demo
https://imgur.com/a/2X8IVMP

### CODE REVIEW

File: src/components/Widgets/Advert/index.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Grid, Image } from 'semantic-ui-react';
+
+export default function index() {
+  return (
+    <div>
```
Comment (codrex) [inline src/components/Widgets/Advert/index.jsx]: Why do we have this extra `div`

File: src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Card, Feed } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import Feeds from './Feeds';
+import './widget.scss';
+
+export default function ArticlesFeedContainer({ description, title, className }) {
+  // const { description, title, className } = props;
```
Comment (codrex) [inline src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx]: remove dead code

File: src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <div>
```
Comment (codrex) [inline src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx]: extra `div` why?

File: src/components/Widgets/Articlewidgets/Feeds.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <ArticlesIcons />
+      </Feed.Content>
+    </Feed.Event>
+  );
+}
+
+MainFeaturedArticle.defaultProps = {
+  description: 'placeholder descritption',
+  title: 'placeholder title',
+  image: 'https://res.cloudinary.com/ah-krypton/image/upload/v1550010452/widget.png'
```
Comment (codrex) [inline src/components/Widgets/Articlewidgets/Feeds.jsx]: move to constants file

File: src/components/Widgets/Articlewidgets/widget.scss [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      -ms-flex-itrem-align: stretch;
+      align-self: stretch;
+      text-align: left;
+      img {
+        border-radius: 0rem;
+      }
+    }
+    width: 100%;
+  }
+  &.grid > {
```
Comment (codrex) [inline src/components/Widgets/Articlewidgets/widget.scss]: Look into this

Comment (ibraheemkabir) [reply src/components/Widgets/Articlewidgets/widget.scss]: i have fixed this

File: src/components/Widgets/Articlewidgets/Feeds.jsx (hunk 4 above)
Comment (ibraheemkabir) [reply src/components/Widgets/Articlewidgets/Feeds.jsx]: done

File: src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx (hunk 3 above)
Comment (ibraheemkabir) [reply src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx]: fixed

File: src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx (hunk 2 above)
Comment (ibraheemkabir) [reply src/components/Widgets/Articlewidgets/ArticlesFeedContainer.jsx]: done

File: src/components/Widgets/Advert/index.jsx (hunk 1 above)
Comment (ibraheemkabir) [reply src/components/Widgets/Advert/index.jsx]: fixed

**Pull request merged**